pkg/controller/daemon/controller_test.go

```diff
@@ -187,21 +187,6 @@ func TestOneNodeDaemonLaunchesPod(t *testing.T) {
 	syncAndValidateDaemonSets(t, manager, ds, podControl, 1, 0)
 }
 
-// DaemonSets should not place onto NotReady nodes
```

Comment:
Can you change this to a test that verifies DaemonSets *will* be placed onto NotReady nodes?